sdk/src/Services/DynamoDBv2/Custom/Conversion/SchemaV1.cs

```diff
@@ -468,15 +573,17 @@ protected bool EntriesToCollection(Type targetType, Type elementType, IEnumerabl
     /// A collection converter which reads both sets of collections (sets and lists)
     /// and writes out sets (NS, SS, BS)
     /// </summary>
-#if NET8_0_OR_GREATER
-    [System.Diagnostics.CodeAnalysis.RequiresUnreferencedCode(Amazon.DynamoDBv2.Custom.Internal.InternalConstants.RequiresUnreferencedCodeMessage)]
-#endif
     internal class PrimitiveCollectionConverterV1 : CollectionConverter
     {
         public PrimitiveCollectionConverterV1()
-            : base(Utils.PrimitiveTypes)
+            : base()
         { }
-        
+
+        public override IEnumerable<Type> GetTargetTypes()
+        {
+            return Utils.PrimitiveTypesCollectionsAndArray;
```

Comment:
This new `PrimitiveTypesCollectionAndArray` field used here is the replacement for the deleted `GetTargetTypes` method above.